stopcovid/dialog/aws_lambdas/handle_command.py

```diff
@@ -19,6 +22,7 @@ def _make_inbound_command(record):
     )
 
 
+@rollbar.lambda_function
```

Comment:
guess I should have figured rollbar would have something specifically for lambda

Reply:
looks like it adds some useful metadata